Makefile

```diff
@@ -1,7 +1,7 @@
-NS = spotify
+NS = default
 
 SERVICE_NAME = bigtable-autoscaler
-SERVICE_IMAGE_VERSION = latest
```

Comment:
jw, why not use latest?

Reply:
wouldn't know which image you're using as the running image will always be 'latest'

Reply:
Yeah but the most updated image would always be the latest one right? And you want to use the most recent one? Idk maybe hard coding the version is better in this case

Reply:
yes but without a deploy the image on a pod will also be latest. It could be two different images but both tagged as `latest`

Reply:
version numbers allow you to refer to a specific PR that reflects the changes that are running